### ISSUE TITLE
USB drive support without granting access

### ISSUE THREAD

**Issue by usamarazzakul** [TeamAmaze/AmazeFileManager#2712]
MOST OF ANDROID TV OR ANDROID TV BOX DOES NOT HAVE FILE by Google LLC .
So many of file maneger or apps can not acces the file manager or cannot acces USB Drive or micro SD Card
Please Create a feature that can work like file by Google LLC

**Comment by ajusama648** [comment]
> MOST OF ANDROID TV OR ANDROID TV BOX DOES NOT HAVE FILE by Google LLC .
> So many of file maneger or apps can not acces the file manager or cannot acces USB Drive or micro SD Card
> Please Create a feature that can work like file by Google LLC

**Comment by EmmanuelMess** [comment]
This has USB support, but you need to grant the app access to the USB drive.

**Comment by ajusama648** [comment]
i know that ...but i can't grant the app access to the usb drive because i don't have the file by Google LLC that can do this ...even i downloaded the file by Google LLC ..but it crashg as i open it...

**Comment by EmmanuelMess** [comment]
Do you have a rooted phone?

**Comment by ajusama648** [comment]
nope

**Comment by EmmanuelMess** [comment]
Feasibility of this has to be discussed then, AFAIK Android would not allow unrestricted access to USB.

**Comment by EmmanuelMess** [comment]
@usamarazzakul was this fixed?

**Comment by usamarazzakul** [comment]
> @usamarazzakul was this fixed?

Yes.Android TV upsated android veraion and added files by google for those kind kf problems
